packages/strapi-plugin-upload/services/Upload.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
             : '',
         buffer,
         mime: stream.type,
-        size: (stream.size / 1000).toFixed(2),
+        size: parseFloat((stream.size / 1000).toFixed(2)),
```

Comment:
`parseFloat` shouldn't be required here. the value is already casted :)

Reply:
okey ;)